docs/scos/dev/the-docker-sdk/202212.0/deploy-file/deploy-file-reference-1.0.md

```diff
@@ -136,6 +136,27 @@ environment: 'docker'
 
 ***
 
+### environment-configuration: secrets:
+
+Defines secrets and their configuration:
+
+* `environment-configuration: secrets: name:`—secret's name.
+* `environment-configuration: secrets: grant:`—users' permissions. `limited` provides read while `public` provides read and write permissions. The default one is `limited`.
```

Comment:
Starting a sentence with a variable name or value is not recommended. I would rephrase it as follows:
```suggestion
* `environment-configuration: secrets: grant:`—users' permissions. The `limited` permission provides read while `public` provides read and write permissions. The default one is `limited`.
```